README.md

```diff
@@ -3,7 +3,19 @@
 This repository contains the source code for a Java package `CedarJava` that supports using the [Cedar](https://www.cedarpolicy.com) policy language.
 It also contains source code for a Rust crate `CedarJavaFFI` that enables calling Cedar library functions (written in Rust) from Java.
 
-You can find build instructions and more information in the subfolders.
+
+## Getting Started
+You can find detailed build instructions and more information in the subfolders.
```

Comment:
Is it easy to add links to them here?

Reply:
Do you mean links to the local files or the files on github?

Reply:
Yes.